src/graph/s21_graph.cc

```diff
@@ -0,0 +1,126 @@
+#include "s21_graph.h"
+
+std::string s21::VertexToString(const Vertex& vertex) {
+  if (std::holds_alternative<int>(vertex)) {
+    return std::to_string(std::get<int>(vertex));
+  } else {
+    return std::get<std::string>(vertex);
+  }
+}
+
+void s21::Graph::LoadGraphFromFile(std::string filename) {
+  std::ifstream in(filename);
+  std::string line;
+
+  if (!in.is_open()) {
+    throw std::runtime_error("Could not open the file.");
+  }
```

Comment:
nit: можно включить в исключение полезную информацию, например, файл с каким именем не удалось открыть.

```suggestion
  if (!in.is_open()) {
    throw std::runtime_error("Could not open the file " + filename);
  }
```